linux_os/guide/services/obsolete/r_services/no_rsh_trust_files/oval/shared.xml

```diff
@@ -7,26 +7,26 @@
       <criterion test_ref="test_no_rsh_trust_files_etc" negate="true" />
     </criteria>
   </definition>
-  <unix:file_test check="all" check_existence="at_least_one_exists" comment="look for .rhosts or .shosts in /root" id="test_no_rsh_trust_files_root" version="1">
+  <unix:file_test check="all" check_existence="at_least_one_exists" comment="look for .rhosts in /root" id="test_no_rsh_trust_files_root" version="1">
     <unix:object object_ref="object_no_rsh_trust_files_root" />
   </unix:file_test>
-  <unix:file_test check="all" check_existence="at_least_one_exists" comment="look for .rhosts or .shosts in /home" id="test_no_rsh_trust_files_home" version="1">
+  <unix:file_test check="all" check_existence="at_least_one_exists" comment="look for .rhosts in /home" id="test_no_rsh_trust_files_home" version="1">
     <unix:object object_ref="object_no_rsh_trust_files_home" />
   </unix:file_test>
-  <unix:file_test check="all" check_existence="at_least_one_exists" comment="look for /etc/hosts.equiv or /etc/shosts.equiv" id="test_no_rsh_trust_files_etc" version="1">
+  <unix:file_test check="all" check_existence="at_least_one_exists" comment="look for /etc/hosts.equiv" id="test_no_rsh_trust_files_etc" version="1">
     <unix:object object_ref="object_no_rsh_trust_files_etc" />
   </unix:file_test>
-  <unix:file_object comment="look for .rhosts or .shosts in /root" id="object_no_rsh_trust_files_root" version="1">
+  <unix:file_object comment="look for .rhosts in /root" id="object_no_rsh_trust_files_root" version="1">
```

Comment:
I am just not sure about this part? Is it enough to look for `.rhosts` files just within `/root` and `/home` or should we search the whole filesystem?

Reply:
The `.rhosts` file are used by the "r-commands' and are effective when present on the user's home directory of the users.
https://www.mkssoftware.com/docs/man4/rhosts.4.asp